packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_list.dart

```diff
@@ -73,7 +73,7 @@ class _ListControlPane extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder<bool>(
-      valueListenable: controller.isTakingSnapshot,
+      valueListenable: controller.isAddingSnapshot,
       builder: (_, isProcessing, __) {
```

Comment:
rename the variable here to match the name of the valueListenable above (e.g. "adding")